client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -40,6 +70,20 @@ class RelatedPostsEdit extends Component {
 		const displayPosts = postsToDisplay.slice( 0, postsToShow );
 		const previewClassName = 'related-posts__preview';
 
+		const displayMockupPosts = posts.length ? false : true;
+
+		const inlineMockupPosts = [];
+		for ( let i = 0; i < postsToShow; i++ ) {
```

Comment:
it looks like `postsToShow` is an array here but we're comparing an integer to it, so I think that means we'll get one iteration since `0` is less than any non-empty array in comparison?

only noting here because it looks confusing from the code

Reply:
`postsToShow` is an integer — the number of posts to display. Perhaps we should change this to something like `postShowCount`?

Reply:
ah my bad - I thought I saw `postsToShow` as the recipient of the `slice` above that `displayPosts` is.

to that end feel free to keep the `for` loop or alternatively consider using `range()`

```js
const inlineMockupPosts = range( postsToShow ).map( key => <MockupPostEdit { ...{ key, display… } } /> )
```

not a big deal either way